Scripts/test.sh

```diff
@@ -23,4 +23,16 @@ struct Secrets {
 }
 EOF
 
-set -o pipefail && xcodebuild test -scheme "ably-asset-tracking-swift-Package" -destination 'platform=iOS Simulator,name=iPhone 12' | xcpretty
+set +e
```

Comment:
Perhaps some commentary above this line to explain why you need to do this. Perhaps also remove the blank line below this line, and also the blank line above the matching `set -e` lower down, in order to visually group them. 🤔

Reply:
Lovely. Thanks!